openc3/lib/openc3/packets/commands.rb

```diff
@@ -308,12 +308,21 @@ def set_parameters(command, params, range_checking)
         item = command.get_item(item_upcase)
         range_check_value = value
 
-        # Convert from state to value if possible
-        if item.states and item.states[value.to_s.upcase]
-          range_check_value = item.states[value.to_s.upcase]
-        end
-
         if range_checking
+          if item.states
+            if item.states[value.to_s.upcase]
+              range_check_value = item.states[value.to_s.upcase]
+            else
+              if command.raw
+                unless item.states.values.include?(value)
```

Comment:
We should allow raw or converted states unless raw.  If raw then only should accept raw states.